crates/optimism/rpc/src/eth/receipt.rs

```diff
@@ -85,6 +85,10 @@ pub struct OpReceiptFieldsBuilder {
     pub l1_blob_base_fee: Option<u128>,
     /// The current L1 blob base fee scalar.
     pub l1_blob_base_fee_scalar: Option<u128>,
+    /// The current operator fee scalar.
+    pub operator_fee_scalar: Option<u128>,
+    /// The current operator fee constant.
+    pub operator_fee_constant: Option<u128>,
```

Comment:
will those be added to the rpc receipt?

of not we can ignore those here

Reply:
Yes they are added

Reply:
ah okay, then we need those here:

https://github.com/alloy-rs/op-alloy/blob/d5e41e51d96857635eff251aea69156a32809c92/crates/rpc-types/src/receipt.rs#L141-L143